components/mdns/mdns.c

```diff
@@ -25,6 +25,9 @@
 #include "esp_wifi.h"
 #endif
 
+#if ESP_IDF_VERSION <= ESP_IDF_VERSION_VAL(5, 1, 0)
+#define CONFIG_ESP_WIFI_ENABLED
```

Comment:
I guess that's why the CI keeps failing
```suggestion
#define CONFIG_ESP_WIFI_ENABLED        1
```